lib/phut.rb

```diff
@@ -1,4 +1,5 @@
 require 'phut/root'
+require 'phut/cli'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.